src/components/posts/List/List.js

```diff
@@ -2,23 +2,39 @@ import React from 'react'
 
 import Actions from './List.Actions'
 
-export default ({ destroyPost, user }) => {
+export default ({ currentUserId, destroyPost, user }) => {
   const posts = user.posts.map(post => (
     <div key={post._id} className='card'>
       <div className='card-body'>
         <p className='card-text'>{ post.content }</p>
-        <blockquote className='blockquote mb-0'>
-          <footer className='blockquote-footer'>Was feeling: { post.emotion }</footer>
-        </blockquote>
+        {post.emotion === '' ? 
```

Comment:
Should also test for undefined, or add some validation on so a field cannot be left blank and become undefined